### DISCUSSION TITLE
How to use the package in Yii2?

### DISCUSSION THREAD

**Discussion by shipinbaoku** [imdhemy/laravel-in-app-purchases#126] [Q&A] [answered]
I have a project developed using yii2.
How to use the package in Yii2?

**Comment by imdhemy** [comment, marked as answer]
It's too late, but I'll answer this question for future users/developers:

This package is built on two separate packages under the hood:
- [Google Play Billing](https://github.com/imdhemy/google-play-billing)
- [App Store IAP](https://github.com/imdhemy/appstore-iap)

You can install both of them on your Yii app and use them together.
